app/controllers/admin/event_schedules_controller.rb

```diff
@@ -0,0 +1,35 @@
+module Admin
+  class EventSchedulesController < Admin::BaseController
+    load_and_authorize_resource :event_schedule
+
+    def create
+      if @event_schedule.save
+        render json: { 'status' => 'ok', event_schedule_id: @event_schedule.id }
+      else
+        render json: { 'status' => "The event couldn't be scheduled. #{@event_schedule.errors.full_messages.join('. ')}" }
```

Comment:
The problem here is that you don't use HTTP status to specify that an error happened! See this blog article:
http://travisjeffery.com/b/2012/04/rendering-errors-in-json-with-rails/

This will also point out, that you should use different callbacks for error and success
(what I already commented in the JS files...)